umbra-js/test/Umbra.test.ts

```diff
@@ -446,4 +848,49 @@ describe('Umbra class', () => {
       );
     });
   });
+
+  describe('ParseOverrides without mutation', () => {
```

Comment:
We can shorten the name here to just be the method under test, then the `it` statements function as its spec
```suggestion
  describe('parseOverrides', () => {
```

Reply:
Addressed here: https://github.com/ScopeLift/umbra-protocol/pull/426/commits/d84ba7d45dcfcd86862d083ce96644cd079dbbd7